tools/vera++/scripts/rules/jerry_funcname_space_parentheses.tcl

```diff
@@ -29,6 +29,10 @@ proc check_part_of_the_file {file line_num col_start col_end} {
     if {[regexp {([[:alnum:]][\s]{2,}\()|([[:alnum:]]\()} $line]} {
         report $file $line_num "there should be exactly one space before left parentheses"
     }
+
+    if {[regexp {\)\w} $line]} {
+        report $file $line_num "there should be exactly one space after right parentheses"
+    }
```

Comment:
This addition seems to be an abuse of this rule, which was created -- I guess -- to make sure that function declarations and call sites always have a space between the function name and the parameter list. But as the changes in the other files reveal, we'd like to use it for type cast styling. That does not look right to me.